### PR TITLE
conda-build: Bring support for Azure Blob Storage

### PR DESCRIPTION
**Pull request by jjerphan**
#### Reference Issues/PRs

Follow-up with https://github.com/man-group/ArcticDB/pull/427.

Fixes https://github.com/man-group/ArcticDB/issues/519.

#### What does this implement/fix? How does it work (high level)? Highlight notable design decisions.

Resolve the discrepancy between PyPI and conda-forge.

#### Checklist

<details>
  <summary>
   Checklist for code changes...
  </summary>
 
 - [x] Have you updated the relevant docstrings and documentation?
 - [x] Is this contribution tested against [all ArcticDB's features](../docs/mkdocs/docs/technical/contributing.md)?
 - [x] Do all exceptions introduced raise appropriate [error messages](https://docs.arcticdb.io/error_messages/)?
 - [x] Are API changes highlighted in the PR description?
 - [x] Is the PR labelled as enhancement or bug so it appears in autogenerated release notes?
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,9 +53,7 @@ ArcticDB is designed from the outset to be resilient; there is no single point o
 | --------------------- | - | - |
 | S3                 | ✔️ | ✔️ |
 | LMDB               | ✔️ | ✔️ |
-| Azure Blob Storage | ✔️ | ➖ |
-
-Support for Azure Blob Storage in conda-forge is tracked in [#519](https://github.com/man-group/ArcticDB/issues/519).
+| Azure Blob Storage | ✔️ | ✔️ |
```
Comment (jamesmunro) [inline README.md:56]: This will be visible a bit early.  People won't see it till we get a release out.

**Review by jamesmunro — APPROVED** [review]

**Review by alexowens90 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jjerphan) [reply README.md:56]: Would adapting the comment to mention that the support will be introduced in 3.1.0 (or even in 3.0.0) make sense?

**Comment by jjerphan** [comment]
The commits of this PR has been cherry-picked to branch `v300` so that they will be part of the `v3.0.0` release.

**Comment by phoebusm** [comment]
[Separate PR](https://github.com/man-group/ArcticDB/pull/853) is created for running the azure-related automated test on conda build

**Pull request merged**